app/views/pages/accessibility.md

```diff
@@ -3,60 +3,90 @@ page_title: Accessibility statement for Register trainee teachers
 title: Accessibility statement for Register trainee teachers
 ---
 <p class='govuk-body'>This accessibility statement applies to Register trainee teachers.</p>
-<p class='govuk-body'>This service is run by the Becoming a Teacher team at the Department for Education. We want as many people as possible to be able to use the service. For example, that means users should be able to:</p>
+<p class='govuk-body'>This website is run by the Department for Education. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
 <ul class='govuk-list govuk-list--bullet'>
   <li>change colours, contrast levels and fonts</li>
-  <li>zoom in up to 300% without the text spilling off the screen</li>
+  <li>zoom in up to 400% without the text spilling off the screen</li>
   <li>navigate most of the service using just a keyboard</li>
   <li>navigate most of the service using speech recognition software</li>
   <li>listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
 </ul>
 <p class='govuk-body'>We’ve also made the text on the service as simple as possible to understand.</p>
-<p class='govuk-body'><a href='https://mcmw.abilitynet.org.uk/' class='govuk-link'>AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+<p class='govuk-body'><a href='https://mcmw.abilitynet.org.uk/' class='govuk-link' target="_blank">AbilityNet (opens in a new tab)</a> has advice on making your device easier to use if you have a disability.</p>
```

Comment:
Add the `rel="noreferrer noopener"` as suggested by the [design system](https://design-system.service.gov.uk/styles/links/#external-links).

```suggestion
<p class='govuk-body'><a href='https://mcmw.abilitynet.org.uk/' class='govuk-link'  rel='noreferrer noopener' target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.</p>
```